src/internal/xml-parser.ts

```diff
@@ -4,6 +4,7 @@ import { XMLParser } from 'fast-xml-parser'
 
 import * as errors from '../errors.ts'
 import type { BucketItemWithMetadata } from '../minio'
+import type { BucketItemFromList } from '../minio'
```

Comment:
Maybe should move this type to internal/type.ts

Reply:
looks like it is already part of `internal/type.ts`. so just update the import